interface_tester/collector.py

```diff
@@ -61,7 +62,7 @@ def __hash__(self):
 
 
 class _CharmsDotYamlSpec(TypedDict):
-    """Specification of the `charms.yaml` file each interface/version dir should contain."""
+    """Specification of the `interface.yaml` file each interface/version dir should contain."""
```

Comment:
Would it make sense to rename the class too?

Unless I've misunderstood something...